tensorflow_recommenders_addons/dynamic_embedding/python/ops/cuckoo_hashtable_ops.py

```diff
@@ -338,6 +338,53 @@ def export(self, name=None):
             self.resource_handle, self._key_dtype, self._value_dtype)
     return keys, values
 
+  def save(self, filepath, buffer_size=4194304, name=None):
```

Comment:
`save_to_file` would be better
Because`save_to_hdfs` is possible in the future.

Reply:
Accept